kolibri/plugins/learn/assets/src/vue/learn-page/allcontent.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+
+  <card-grid :header="'All Content'" v-if="all.content.length">
+
+    <div slot="headerbox" class="allnav">
+
```

Comment:
1. I would also recommend adding the `id` to the `All Content` heading above and  `role="navigation" aria-labelledby="id-of-the-h2"` for this `div`.

2. When TAB-navigating and choosing either `Previous` or `Next` links here, the focus jumps back above, so user has to tab through the header with the channel selector, search button, `All Content` prev-next links to get to the actual content item. Can we set the focus to the first item in the `card-grid` upon selecting `Previous` or `Next` links?